.buildkite/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+mkdir build_src
+
+if [[ $LE_TRIGGERED_FROM_BUILD_ID ]]
```

Comment:
This is part of the buildkite build infrastructure I've been working on over the past couple months. Each repo has a pipeline with this defined - I'm not expecting many people to venture into the `.buildkite` directory without knowing what they're doing, and things are still in enough flux that I'd rather not document this in a comment for every repo only to have to change it later. 

The pipeline can be triggered by Kolibri's pipeline. Kolibri's pipeline sets this envvar up for every installer's pipeline trigger. The build ID Kolibri's build UID, assigned by Buildkite. Passing it along allows this pipeline to know what "parent" build triggered it and to download the appropriate WHL from that pipeline.

Reply:
Thanks for the explanation :pray: 

When putting code in different repositories, it's nice to have comments to explain where the environment originates from. Otherwise it takes more time for others to quick assess what it is and what it does. You can always briefly refer to an external documentation, it's just for the reader to have a fair chance to understand where to go next?

> Environment inherited from external Buildkite pipeline defined in: https://github.com/learningequality/blah

Reply:
I'll consider it, dude. I hear what you're saying. But like I said above: things are still in flux, and no one who doesn't know what this line is doing should be messing with it. 

Most of the team is aware that the build pipeline is being restructured into a multi-repo contraption, and makes a point to avoid touching these parts as they know it's under active dev and part of larger ecosystem.